src/app/core/analytics/matomo.service.ts

```diff
@@ -0,0 +1,38 @@
+import { Injectable } from "@angular/core";
+
+/**
+ * This class dynamically adds sets up everything for Matomo.
+ *
+ * The code is inspired by:
+ * https://github.com/Arnaud73/ngx-matomo/blob/master/projects/ngx-matomo/src/lib/matomo-injector.service.ts
+ */
+@Injectable({
+  providedIn: "root",
+})
+export class MatomoService {
```

Comment:
I think this `MatomoService` should be part of the existing `AnalyticsService`, whose purpose is to manage everything related to usage analytics (in our case with Matomo as the backend)